bin/foreman-maintain

```diff
@@ -4,6 +4,11 @@ $LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
 
 require 'foreman_maintain'
 
+CONFIG_FILE = '/etc/foreman-maintain/config/foreman_maintain.yml'.freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data.yml'.freeze
```

Comment:
@iNecas , 
I will go with `foreman_maintain.yml.packaging` option. 
Previously I thought that user have to create foreman_maintain.yml from .yml.example file that's why I added these constants additionally. 
I will modify code and update the PR for review.